core/src/kmx/kmx_file.h

```diff
@@ -340,11 +340,14 @@ struct COMP_KEYBOARD {
   KMX_DWORD dwBitmapSize;   // 003C size in bytes of the bitmaps
 };
 
-// TODO-LDML: fill this in
+/**
+ * Only valid if comp_keyboard.dwFlags&0x20
```

Comment:
```suggestion
 * Only valid if comp_keyboard.dwFlags&KF_KMXPLUS
```

Feel like it's better docs to use the const name than the value.